readme.txt

```diff
@@ -1,17 +1,17 @@
-=== PayEx WooCommerce Payments ===
+=== Swedbank Pay WooCommerce Payments ===
```

Comment:
`readme.txt` is just repetition of `readme.md` and can be deleted.

Reply:
Wordpress Plugin catalog requires readme.txt